assets/source/setup-guide/app/style.scss

```
@@ -345,40 +300,18 @@ $root: ".woocommerce-setup-guide";
 
 				.components-card__body {
 
-					> *:not(:last-child) {
-						margin-bottom: 16px;
-					}
-				}
-
-				#{$root} {
-
-					&__step {
-
-						&-status {
-							padding: 0;
-						}
+					> *:not(:first-child) {
+						margin-bottom: var(--main-gap);
 					}
-				}
-
-				.is-verified {
 
-					> * {
-						margin: 8px 0;
-					}
-				}
-
-				.has-idle {
-
-					#{$root} {
-
-						&__step-status__icon {
-							display: none;
-						}
+					> *:first-child {
```

💅  `*` is not necessary.

```suggestion
					> :first-child {
```